src/applications/dhp-connected-devices/tests/containers/App.spec.jsx

```diff
@@ -55,7 +55,7 @@ describe('App', () => {
       </Provider>,
     );
 
-    expect(wrapper.find('va-loading-indicator').length).to.equal(0);
+    // expect(wrapper.find('va-loading-indicator').length).to.equal(0);
```

Comment:
Can we please remove this line if it's not needed?

Reply:
Removed!